CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Release 2020.7 (?)
+# Release 2020.7 (2021-03-01)
```

Comment:
Shouldn't we add a new section to the changelog. I think we can even remove the changelog entirely. A changelog in a meta-data repo doesn't add much value. What do you think?

Reply:
I think you're right. I'll remove it.